.circleci/config.yml

```
@@ -135,114 +141,117 @@ references:
+                      fi
+          - *calypso_prepare_cache
+          - *calypso_restore_cache
+          - *restore_nvm
+          - *setup_nvm
+          - *save_nvm
+          - run:
+              name: Decrypt assets
+              command: |
+                      openssl aes-256-cbc -md md5 -d -in resource/calypso/secrets.json.enc -out calypso/config/secrets.json -k "${CALYPSO_SECRETS_ENCRYPTION_KEY}"
+                      openssl aes-256-cbc -md md5 -d -in resource/certificates/mac.p12.enc -out resource/certificates/mac.p12 -k "${CALYPSO_SECRETS_ENCRYPTION_KEY}"
+                      openssl aes-256-cbc -md md5 -d -in resource/certificates/win.p12.enc -out resource/certificates/win.p12 -k "${CALYPSO_SECRETS_ENCRYPTION_KEY}"
+          - *npm_restore_cache
+          - run:
+              name: Npm install
+              command: |
+                      source $HOME/.nvm/nvm.sh
+                      nvm use
+                      npm ci
+                      cd calypso
+                      npm ci
+          - *npm_save_cache
+          - run:
+              name: Build sources
+              no_output_timeout: 20m
+              command: |
+                      source $HOME/.nvm/nvm.sh
+                      nvm use
 
+                      # only build the whole bundle when there is no calypso-hash file
+                      if [ "$(cat calypso-current-hash)" != "$(cat calypso-hash)" ]; then
+                        make build-source CONFIG_ENV=$CONFIG_ENV EMIT_STATS=false MINIFY_JS=$MINIFY_JS NODE_ARGS="$NODE_ARGS"
+                      else
+                        make desktop/config.json CONFIG_ENV=$CONFIG_ENV
+                        make build-desktop
+                      fi
+          - run:
+              name: Test
+              command: |
+                      set +e
 
-                    if [ -n "${CALYPSO_HASH}" ]; then
-                      CONFIG_ENV=test
-                    fi
+                      # At this stage, we can only do canary tests when CONFIG_ENV=test as the electron binary is not signed
+                      # TODO: We might be able to ignore this once we switched to electron-builders auto-update
+                      if [ "$CONFIG_ENV" == "test" ]; then
+                        source $HOME/.nvm/nvm.sh
+                        nvm use
+                        make test
+                      fi
+          - *calypso_finalize_cache
+          - *calypso_save_cache
 
-                    make build-source CONFIG_ENV=$CONFIG_ENV EMIT_STATS=true
-                  else
-                    make desktop/config.json CONFIG_ENV=$CONFIG_ENV
-                    make build-desktop
-                  fi
-      - run:
-          name: Test
-          command: |
-                  set +e
+jobs:
+  build:
+    docker:
+      - image: circleci/node:10.14.0-browsers
+    environment:
+      CONFIG_ENV: test
+      MINIFY_JS: false
+      NODE_ARGS: --max_old_space_size=2048
+    steps:
```

Is this missing from this job?
```suggestion
    working_directory: ~/wp-desktop	
    steps:
```

---

👍  I guess since we specify this in every other job its no harm to be specific. Without it it defaults to `~/project` which works fine but consistency is better.